api/routes/resource.routes.js

```diff
@@ -131,7 +133,7 @@ module.exports = (app) => {
         });
         return res.json(resourceUtil.toJSON(newResource));
       } catch (err) {
-        res.json({ errors: [{ msg: '' + err }] });
+        res.json({ errors: [{ msg: '' + err }], body: req.body });
```

Comment:
Is this for testing? should it be removed?

Reply:
opps, yup